### DISCUSSION TITLE
ignore "main" in package.json?

### DISCUSSION THREAD

**Discussion by ELI7VH** [parcel-bundler/parcel#6172] [Q&A]
hello. I am writing a library and using parcel in my dev environement. 

I am trying to build some demos, but I use "main" in my package.json, and it's now breaking my parcel build.

Is there any way to ignore this "main" for when I am trying to build my demos using `parcel build ...`

**Comment by mischnic** [comment]
Yes: https://v2.parceljs.org/configuration/package-json/#main-%2F-module-%2F-browser

```json
{
  "main": "unrelated.js",
  "targets": {
    "main": false
  }
}
```
